# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/moz.build
# Commit: 2b647eb982bc
# Full Hash: 2b647eb982bc0068d39fe58c6f59bebe55909343
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-07 21:43:58
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 7c11166969ff -r 2b647eb982bc security/manager/ssl/moz.build
--- a/security/manager/ssl/moz.build	Wed Jun 07 15:45:14 2023 +0000
+++ b/security/manager/ssl/moz.build	Wed Jun 07 15:56:37 2023 +0000
@@ -113,7 +113,6 @@
     "nsCertOverrideService.cpp",
     "nsCertTree.cpp",
     "nsClientAuthRemember.cpp",
-    "nsCryptoHash.cpp",
     "nsNSSCallbacks.cpp",
     "nsNSSCertHelper.cpp",
     "nsNSSCertificate.cpp",